invenio_users_resources/entity_resolvers.py

```diff
@@ -63,17 +63,27 @@ def pick_resolved_fields(self, identity, resolved_dict):
         avatar = current_users_service.links_item_tpl.expand(identity, fake_user_obj)[
             "avatar"
         ]
-        return {
+        obj = {
+            "identity": identity,
             "id": resolved_dict["id"],
-            "username": resolved_dict["username"],
+            "username": resolved_dict.get("username", ""),
```

Comment:
@TLGINO Could you add a screenshot of how would the request be displayed now? What would appear in the `username` place? a placeholder? would it be empty? If the later, what do you think about defaulting to a mandatory field? (e.g. email, or full_name?)